Sources/ProjectSpec/Target.swift

```diff
@@ -13,6 +13,11 @@ public struct Target {
     public var postbuildScripts: [BuildScript]
     public var configFiles: [String: String]
     public var scheme: TargetScheme?
+    
+    public var buildToolPath: String?
```

Comment:
As these won't be used most of the time, can we extract them into a nullable `legacy: LegacyTarget` struct

Reply:
I like that!